### ISSUE TITLE
EU region

### ISSUE THREAD

**Issue by Alathiel** [NotHunter101/ExtraRolesAmongUs#13]
If i try to create a lobby in EU region i get the error

"Disconnected from the server.
Reliable packet 1 (size=104) was not ack'd after 7531ms (9 resends)"

I tried North America and Asia and it works.

**Comment by Erumeldor** [comment]
I can second that. Not the exact error message (because that's just a general error), but the fact that it is kind of impossible to play on EU.
I know that EU servers are more unstable than NA, but in normal mode you can play.

With the mod we cannot play an hour without the whole lobby crashing.

**Comment by xmichi2000** [comment]
i'm experiencing the same

EU seems to be hella unstable with ExtraRoles for some reason

**Comment by NotHunter101** [comment]
Not sure why this happens, to be honest. The current workaround is just using NA servers.

**Comment by jpicht** [comment]
That has nothing to do with the mod - the EU servers have never reliably worked for our friend group.

**Comment by Erumeldor** [comment]
> That has nothing to do with the mod - the EU servers have never reliably worked for our friend group.

That's wrong. Yes, EU servers still do have some bugs, like problems connecting (the thing where you have to join a public game first before being able to join the private lobby). And a few months ago it was impossible in the EU evening hours to get a lobby starting. But that's not the case anymore.

With our pure german group it is much better to play on EU for us that on NA. We often have super annoying desync problems on NA, sneaky kills are impossible to do.

**Comment by jpicht** [comment]
> > That has nothing to do with the mod - the EU servers have never reliably worked for our friend group.
> 
> That's wrong. Yes, EU servers still do have some bugs, like problems connecting (the thing where you have to join a public game first before being able to join the private lobby). And a few months ago it was impossible in the EU evening hours to get a lobby starting. But that's not the case anymore.
> 
> With our pure german group it is much better to play on EU for us that on NA. We often have super annoying desync problems on NA, sneaky kills are impossible to do.

I've only started playing the game two month ago and the only times all players were able to join the server, it bugged out and two players were stuck as imposters.

**Comment by MrRedPandabaer** [comment]
I can report the same thing - This has nothing to do with the EU servers. Without the mod: I can join EU. With the mods: It's not possible. Something is up here.

**Comment by NotHunter101** [comment]
Not sure why I waited so long, but today I tried to launch a lobby in the Europe region, and it worked. The fact I cannot reproduce this very obvious and real issue is concerning; I will investigate more.

**Comment by manniL** [comment]
I had no issues either with EU servers today.

The bug mentioned above happened to me though even without mods. Joining a public lobby or switching regions to NA and back to EU usually solves it for me.

**Comment by xmichi2000** [comment]
> Not sure why I waited so long, but today I tried to launch a lobby in the Europe region, and it worked. The fact I cannot reproduce this very obvious and real issue is concerning; I will investigate more.

Yeah. Not sure what you've changed to fix it, but since the new version it works like a charm.

**Comment by Popeye4242** [comment]
I am playing nearly everyday on EU and this seems to be stable since last update.
